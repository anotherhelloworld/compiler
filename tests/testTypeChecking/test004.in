var
    a: integer = 10;
begin
    a := 5 + 5;
end.
