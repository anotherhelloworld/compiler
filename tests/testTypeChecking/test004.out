begin
      a
   :=
         5
      +
         5
end
